lnrpc/routerrpc/router.proto

```diff
@@ -2,42 +2,59 @@ syntax = "proto3";
 
 package routerrpc;
 
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/routerrpc";
+
 message PaymentRequest {
-    /**
-    A serialized BOLT-11 payment request that contains all information
-    required to dispatch the payment. If the pay req is invalid, or expired,
-    an error will be returned.
-    */
-    string pay_req = 1;
+    /// The identity pubkey of the payment recipient
```

Comment:
This information is already contained in the payment request, why do we need to duplicate it?

Reply:
There's no commit body message explaining rationale or motivation.

Reply:
I should rename the message to `SendPaymentRequest`. It is not an encoded invoice.

Reply:
Renamed